src/RuleSet/RuleSet.php

```diff
@@ -271,8 +271,8 @@ protected function renderRules(OutputFormat $outputFormat)
         $result = '';
         $isFirst = true;
         $oNextLevel = $outputFormat->nextLevel();
-        foreach ($this->aRules as $aRules) {
-            foreach ($aRules as $rule) {
+        foreach ($this->rules as $rules) {
+            foreach ($rules as $rule) {
```

Comment:
I'm struggling to understand how and why this can be doubly-iterated over - since `Rule` does not appear to be `Iterable`.

Reply:
I don’t understand this either. Either this is broken (and not tested, or never called), or the code somehow still is working fine. 😉